### DISCUSSION TITLE
[Request] Release Yt Music Also in your telegram Channel

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#273] [General]
Please release Yt music On Your Telegram Channel

**Comment by Degl1** [comment]
Could you elaborate? As ReVanced hasn't published any APK's to Telegram and any other channels doing so are possibly distrobuting malicious ReVanced APK's.

**Reply by ghost** [reply]
This is the telegram Channel which is Publishing These apps https://t.me/ReVancedTeam1

**Reply by Degl1** [reply]
Except this isn't the official channel. This is a third party channel publishing maybe even malicious prebuilt APK's.

**Comment by oSumAtrIX** [comment]
ReVanced will only publish to revanced.app and underlying pages.
